test/test-platform/TestPlatform.cpp

```diff
@@ -369,9 +368,11 @@ bool CTestPlatform::createExclusiveSelectionCriterion(const string& strName,
         ostrValue << "State_";
         ostrValue << uistate;
 
-        if (!pCriterionType->addValuePair(uistate, ostrValue.str())) {
+        if (!pCriterionType->addValuePair(
+                    uistate, ostrValue.str(), strResult)) {
 
-            strResult = "Unable to add value: " + ostrValue.str();
+            strResult = "Unable to add value: "
```

Comment:
It was part of the original pull request and does not really matter (wrapped at 80).